docs/WalletGuide.md

```diff
@@ -40,3 +48,115 @@ val walletCustom = Wallet(StellarConfiguration.Testnet, ApplicationConfiguration
 <!--- INCLUDE
 val wallet = Wallet(StellarConfiguration.Testnet)
 -->
+
+<!--- INCLUDE .*stellar.*
+import org.stellar.walletsdk.*
```

Comment:
FYI: this import shouldn't be necessary (already defined above)

Reply:
Somehow, it's not adding this import from the global INCLUDE.